framework/src/samplers/Sampler.C

```diff
@@ -252,6 +260,7 @@ std::vector<Real>
 Sampler::getNextLocalRow()
 {
   TIME_SECTION(_perf_get_next_local_row);
+  checkReinitStatus();
```

Comment:
Is this needed here? Looks like it is covered by `getLocalRowBegin/End()` and the assert in the column loop.

Reply:
Just being paranoid. For example, someone could change the size in the loop after calling the begin/end functions.